### PR TITLE
[IMP] account.move form: moved company_id to other info tab

### PR DESCRIPTION
**Pull request by bfr-o**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by qdp-odoo** [comment]
@bfr-o can you squash your commits and improve the commit message. I'd sugeest something alike

> [IMP] account: usability of account.move form view
> - moved company_id in other info tab
> - whatever other change you may find useful to explicitly depict
> 
> was task: \<insert here the task ID>

File: addons/purchase/views/account_move_views.xml [hunk 1]
```diff
@@ -9,14 +9,19 @@
             <!-- Auto-complete could be done from either a bill either a purchase order -->
             <field name="invoice_vendor_bill_id" position="after">
                 <field name="purchase_id" invisible="1"/>
-                <field name="purchase_vendor_bill_id"
+                <label for="invoice_vendor_bill_id" string="Auto-Complete" class="oe_edit_only"
+                       attrs="{'invisible': ['|', '|', ('state','not in',['draft']), ('state', '=', 'purchase'), ('move_type', '!=', 'in_invoice')]}" />
```
Comment (qdp-odoo) [inline addons/purchase/views/account_move_views.xml:13]: that's pretty much tordu: \<label for a X field with a string and followed by a field which isn't the X-field>

what's the reason of all this brol?

there has to be a simpler way...

Comment (bfr-o) [reply addons/purchase/views/account_move_views.xml:13]: I know, but if I put class="oe_edit_only" on the field without doing this it doesn't work, I could figure out why.

File: addons/account/views/account_move_views.xml [hunk 2]
```diff
@@ -607,18 +607,21 @@
...
+                                <field name="invoice_payment_ref"
+                                       attrs="{'invisible': ['|',('move_type', 'not in', ('out_invoice', 'out_refund', 'in_invoice', 'in_refund', 'out_receipt', 'in_receipt')), ('state', '=', 'draft')], 'readonly': [('state', '!=', 'draft')]}"/>
+                                <field name="invoice_partner_bank_id"
+                                       context="{'default_partner_id': commercial_partner_id}"
+                                       domain="[('partner_id', '=', bank_partner_id)]"
+                                       attrs="{'invisible': [('move_type', 'not in', ('in_invoice', 'in_refund', 'in_receipt'))], 'readonly': [('state', '!=', 'draft')]}"/>
+                                <label name="invoice_vendor_bill_id_label" for="invoice_vendor_bill_id" string="Auto-Complete" class="oe_edit_only"
+                                       attrs="{'invisible': ['|', ('state', '!=', 'draft'), ('move_type', '!=', 'in_invoice')]}"/>
+                                <field name="invoice_vendor_bill_id" nolabel="1" class="oe_edit_only"
```
Comment (qdp-odoo) [inline addons/account/views/account_move_views.xml:621]: @aab-odoo this is the bidule we talked about

**Comment by qdp-odoo** [comment]
@robodoo r+

**Comment by qdp-odoo** [comment]
@robodoo r+

**Pull request closed without merging**